### DISCUSSION TITLE
deck.gl localization

### DISCUSSION THREAD

**Discussion by obones** [visgl/deck.gl#5328] [Q&A] [answered]
Hello,

Various text elements may be displayed on a map based visualization inside deck.gl and I would like everything to be localized, in particular to French.
I understand that the map tiles and their overlays are provided by a third party (MapTiler, MapBox, Google) so the translations will have to come from them.
But what about strings internal to deck.gl? I searched in the documentation but could not find anything related to localization.

Is it possible? If yes, what are the available languages?
If not, should I create a feature request?

Thanks for your answers

**Comment by Pessimistress** [comment, marked as answer]
I don’t believe deck.gl displays any user-facing text strings. Error/warning messages are logged to the console and can be handled via custom callbacks.

What specifically are you trying to localize?

**Reply by obones** [reply]
I was indeed wondering about user messages, and could not figure out from the code if they would be displayed to users.

Thanks for your help.
